plugins/select/README.md

```diff
@@ -200,6 +198,24 @@ connect(select(models => {
 }))(...)
 ```
 
+Here is a full example combining raw, uncomputed state with selected state:
+
+```js
+const mapStateToProps = state => {
+  const selection = select(models => ({
```

Comment:
You should create the selector outside of mapStateToProps. 

You'll get a lot more performance if your selector is static and the cache is re-used.